nengo_loihi/hardware/nxsdk_shim.py

```diff
@@ -13,6 +15,27 @@
     def assert_nxsdk():
         pass
 
+    from nxsdk.graph import graph
```

Comment:
This might fail with older nxsdk, since they moved a bunch of this stuff around in 0.8.

Reply:
I checked and the Graph class is in the same place in 0.7.5.  However, I didn't actually try running things in 0.7.5 to confirm it works.